### DISCUSSION TITLE
Problem with line-height

### DISCUSSION THREAD

**Discussion by ballotade** [dompdf/dompdf#2973] [Q&A] [answered]
Hello again,

my project with dompdf ist growing very well, but now I have a problem with line-height.
The text in the pdf ist much lower than in the html.

I found an older discussion where they talk about change the DOMPDF_FONT_HEIGHT_RATIO.
Where can I change it? 
Or is there another solution?

I'm using dompdf 2.0.
Thanks :)

**Comment by bsweeney** [comment, marked as answer]
In Dompdf 2.0 that would be the `fontHeightRatio` [Option](https://github.com/dompdf/dompdf/wiki/Usage#options), which is 1.1 by default.

Dompdf should be fairly close as far as line height now if you want to share some sample HTML that shows the issue we can see if there's an issue to address.

**Reply by ballotade** [reply]
Thank you. :)
$dompdf->set_option('fontHeightRatio', 0.9) was the right value.
